### DISCUSSION TITLE
How to use code interpreter in autogen studio?

### DISCUSSION THREAD

**Discussion by easontsai** [microsoft/autogen#2443] [Q&A]
I follow the tutorial in this post.
[https://microsoft.github.io/autogen/blog/2023/11/13/OAI-assistants#code-interpreter-example](url)

any sample to integrate with autogen studio?

Thanks.

**Comment by ekzhu** [comment]
Our roadmap for AutoGen Studio: https://github.com/microsoft/autogen/issues/737. cc @victordibia for possible OAI assistant integration.
